src/index.tsx

```diff
@@ -4,19 +4,15 @@ import { HashRouter as Router } from 'react-router-dom';
 
 import { store } from './app/store';
```

Comment:
Ensure that the `store` is correctly configured and imported from the `app/store` file.